geomesa-tools-common/src/main/scala/org/locationtech/geomesa/tools/common/CommonParams.scala

```diff
@@ -64,12 +64,12 @@ trait OptionalDTGParam {
 
 trait OptionalAttributesParam {
   @Parameter(names = Array("-a", "--attributes"), description = "Attributes to evaluate (comma-separated)")
-  var attributes: java.util.List[String] = null
+  var attributes: String = null
```

Comment:
is there a reason to change this to a string? i believe (although not 100% sure) keeping it a list lets you specify multiple flags instead of forcing comma-separated, e.g. `-a foo -a bar` vs `-a foo,bar`. But jcommander also does some splitting on a single argument if you specify a list - i think we override that in certain places. anyway, not sure it really matters